Prowl.Editor/EditorWindows/MainMenuItems.cs

```diff
@@ -16,7 +16,7 @@ public static class MainMenuItems
         [MenuItem("Create/Folder")]
         public static void CreateFolder()
         {
-            Directory ??= new DirectoryInfo(Project.ProjectAssetDirectory);
```

Comment:
Hmm, these null checks here were placed as a fallback when no AssetBrowserWindow has been created.
AssetBrowserWindow.CurDirectory can be null
It also should support when right-clicking a folder and pressing Create Material, it should create it inside the folder not the directory I'm inside.

Reply:
The problem is further up, Whatever is calling MenuItem.DrawMenuRoot("Create");
Isnt setting the MainMenuItems.Directory properly, which at a first glance looks like thats the case in a few issues.

Reply:
okok, I get it now.